src/btree/bt_cursor.c

```diff
@@ -73,6 +66,9 @@ __btcur_bounds_search_near_reposition(WT_SESSION_IMPL *session, WT_CURSOR_BTREE
 
     WT_ASSERT(session, WT_CURSOR_BOUNDS_SET(cursor));
 
+    if (CUR2BT(cursor)->type != BTREE_ROW)
```

Comment:
Interesting addition, did this get added as a result of testing or otherwise?

Reply:
I added this for now because search_near isn't supported yet. I added this just as an extra measure incase developers run search_near() tests and it suddenly doesn't work

Reply:
Right because we've removed the higher level check that blocked column store originally, gotcha.